database/migrations/2022_12_10_183251_increment_user_i_ds.php

```diff
@@ -26,7 +26,7 @@ public function up(): void
 		foreach (DB::table('users')->orderByDesc('id')->get() as $user) {
 			$oldID = $user->id;
 			$newID = $oldID + 1;
-			DB::table('users')->find($oldID)->update(['id' => $newID]);
```

Comment:
find was returning an `object` without methods. Resulting to this error:
`Call to undefined method stdClass::update()`